### DISCUSSION TITLE
The node can't read the content of some mail

### DISCUSSION THREAD

**Discussion by cocomongo4** [umanamente/n8n-nodes-imap#11] [Q&A] [answered]
Hello, thank for your work the node works well, but I've run into a problem.

For approximately 80% of the received emails, the node works perfectly. However, for the other 20%, the node can't return the text content or the HTML content of the email. I have access to some basic information like the title, the recipient, and the sender, but that's all.

I have tried creating another workflow, resetting the node, and updating it, but I can't seem to get it to work.

**Comment by umanamente** [comment]
Could you please enable **textContent**, **htmlContent**, **attachmentsInfo**, **bodyStructure** options in the "Get Emails" and paste here the json of the email with this problem?

**Comment by cocomongo4** [comment]
Thanks for your answer . Here is the JSON  :

[
  {
    "seq": 1,
    "uid": 406,
    "modseq": "644",
    "bodyStructure": {
      "type": "text/html",
      "parameters": {
        "charset": "UTF-8"
      },
      "encoding": "quoted-printable",
      "size": 22984,
      "lineCount": 498
    },
    "envelope": {
      "date": "2024-04-18T20:24:11.000Z",
      "subject": "Re: Update",
      "from": [
        {
          "name": "",
          "address": "khoujane@gmx.fr"
        }
      ],
      "sender": [
        {
          "name": "",
          "address": "khoujane@gmx.fr"
        }
      ],
      "replyTo": [
        {
          "name": "",
          "address": "khoujane@gmx.fr"
        }
      ],
      "to": [
        {
          "name": "",
          "address": "info@trainelectrique.com"
        }
      ],
      "inReplyTo": "<6bbea0af-3714-6a0f-ba59-7ca2a314e2e2@trainelectrique.com>",
      "messageId": "<trinity-84cd2dac-fa13-4bbe-9b02-a36c0d3187b2-1703487251055@msvc-mesg-gmx005>"
    },
    "id": "2023af24b387e9be10c495ef4ecf8918",
    "attachmentsInfo": []
  }
]




![Untitled1](https://github.com/umanamente/n8n-nodes-imap/assets/166753127/9932ab92-6a3c-4781-bb53-496033fcf96e)

**Comment by umanamente** [comment, marked as answer]
Fixed in [v1.3.0](https://github.com/umanamente/n8n-nodes-imap/releases/tag/v1.3.0)

**Comment by cocomongo4** [comment]
It work well now thanks !
